userland/AppMakefile.mk

```diff
@@ -66,19 +100,19 @@ $$(BUILDDIR)/$(1):
 
 # First step doesn't actually compile, just generate header dependency information
 # More info on our approach here: http://stackoverflow.com/questions/97338
-$$(BUILDDIR)/$(1)/%.o: %.c | $$(BUILDDIR)/$(1)
+$$(BUILDDIR)/$(1)/%.o: %.c | $$(BUILDDIR)/$(1) $$(HDRS)
```

Comment:
I'm still confused what's going on with it. As far as I can tell it is only used as a requirement that they exist for several build targets. They aren't included in the rules though.